docs/src/markdown/installation.md

```diff
@@ -22,10 +22,15 @@ Name                             | Details
     ```
 
     Replace `python2.7-dev` with the Python version you are using.
+    
+    For Fedora 26, you need few dependecies to built wxPython:
+    ```bash
+    sudo dnf install gcc-c++ wxGTK-devel gstreamer-devel webkitgtk-devel GConf2-devel gstreamer-plugins-base-devel
+    ```
 
-    If your Linux distribution has `gstreamer` 1.0 available, you can install the dev packages for that instead of the 0.10 version.
+    If your Linux distribution has `gstreamer` 1.0 available, you can install the dev packages (like Fedora distro) for that instead of the 0.10 version.
```

Comment:
Please place `(like the Fedora distro)` before the comma in the sentence as is qualifies that part of the sentence.

Reply:
Fixed on the latest commit